Packs/EWS/Integrations/EWSv2/EWSv2.py

```diff
@@ -1115,81 +1151,54 @@ def parse_incident_from_item(item, is_fetch):
-                labels.append({'type': label_attachment_id_type, 'value': attachment.attachment_id.id})
-
-    # handle headers
-    if item.headers:
-        headers = []
-        for header in item.headers:
-            labels.append({'type': 'Email/Header/{}'.format(header.name), 'value': str(header.value)})
-            headers.append("{}: {}".format(header.name, header.value))
-        labels.append({'type': 'Email/headers', 'value': "\r\n".join(headers)})
-
-    # handle item id
-    if item.message_id:
-        labels.append({'type': 'Email/MessageId', 'value': str(item.message_id)})
-
-    if item.item_id:
-        labels.append({'type': 'Email/ID', 'value': item.item_id})
-        labels.append({'type': 'Email/itemId', 'value': item.item_id})
-
-    # handle conversion id
-    if item.conversation_id:
-        labels.append({'type': 'Email/ConversionID', 'value': item.conversation_id.id})
-
-    if MARK_AS_READ and is_fetch:
-        item.is_read = True
-        try:
-            item.save()
-        except ErrorIrresolvableConflict:
-            time.sleep(0.5)
-            item.save()
+                        else:
+                            incident['attachment'].append({
+                                'name': get_attachment_name(attachment.name) + ".eml"
+                            })
+
+                    labels.append({'type': label_attachment_type, 'value': get_attachment_name(attachment.name)})
+                    labels.append({'type': label_attachment_id_type, 'value': attachment.attachment_id.id})
+
+        # handle headers
+        if item.headers:
+            headers = []
+            for header in item.headers:
+                labels.append({'type': 'Email/Header/{}'.format(header.name), 'value': str(header.value)})
+                headers.append("{}: {}".format(header.name, header.value))
+            labels.append({'type': 'Email/headers', 'value': "\r\n".join(headers)})
+
+        # handle item id
+        if item.message_id:
+            labels.append({'type': 'Email/MessageId', 'value': str(item.message_id)})
+
+        if item.item_id:
+            labels.append({'type': 'Email/ID', 'value': item.item_id})
+            labels.append({'type': 'Email/itemId', 'value': item.item_id})
 
-    incident['labels'] = labels
-    incident['rawJSON'] = json.dumps(parse_item_as_dict(item, None), ensure_ascii=False)
+        # handle conversion id
+        if item.conversation_id:
+            labels.append({'type': 'Email/ConversionID', 'value': item.conversation_id.id})
+
+        if MARK_AS_READ and is_fetch:
+            item.is_read = True
+            try:
+                item.save()
+            except ErrorIrresolvableConflict:
+                time.sleep(0.5)
+                item.save()
+
+        incident['labels'] = labels
+        incident['rawJSON'] = json.dumps(parse_item_as_dict(item, None), ensure_ascii=False)
+
+    except Exception as e:
+        if 'Message is not decoded yet' in str(e):
```

Comment:
so we are just skipping them? can't we add them in a raw format and then add the password?